igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -483,6 +769,14 @@
     "### 4.1 Effect of Water Masking"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4d078c0c-e13f-4a42-8bec-192e791c25d9",
+   "metadata": {},
+   "source": [
+    "The first thing that we will take a look at is a real problem that Grapenthin et al. had to troubleshoot when they were getting ready to publish their analysis. In the first version of their analysis, the water mask option HyP3 provided did not mask the water within 100 meters of the land surface. Even this relatively small amount of water present in the scene led to an area of erroneous subsidence northeast Mt. Edgecumbe. The HyP3 team has now tightened their water mask so that it now follows the coastline closely, but we can re-create this issue by comparing the water masked and non water masked versions of our descending track stack."
```

Comment:
```suggestion
    "The first thing that we will take a look at is a real problem that Grapenthin et al. had to troubleshoot when they were getting ready to publish their analysis. In the first version of their analysis, the water mask option HyP3 provided did not mask the water within 100 meters of the land surface. Even this relatively small amount of water present in the scene led to an area of erroneous subsidence northeast Mt. Edgecumbe. The HyP3 team has now tightened their water mask so that it follows the coastline closely, but we can re-create this issue by comparing the water masked and non water masked versions of our descending track stack."
```